cfme/base/__init__.py

```diff
@@ -132,6 +132,10 @@ def upload_custom_logo(self, file_type, file_data=None, enable=True):
         view.customlogos.save_button.click()
         view.flash.assert_no_error()
 
+    @property
```

Comment:
Consider using VersionPicker here, and defining it as a class attribute instead of a property method.  Then there isn't an explicit call to self.appliance.version, it'll pick itself when referenced.

```
intel_name = VersionPicker({Version.LATEST: 'Overview', Version.LOWEST: 'Cloud Intel'})
```